src/v2/Apps/Partner/Components/PartnerArtists/PartnerArtistDetailsList.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box } from "@artsy/palette"
+import React from "react"
+import { useSystemContext } from "v2/Artsy"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { PartnerArtistDetailsFragmentContainer } from "../../Components/PartnerArtists"
+import { PartnerArtistDetailsList_edges } from "v2/__generated__/PartnerArtistDetailsList_edges.graphql"
+import { PartnerArtistDetailsListQuery } from "v2/__generated__/PartnerArtistDetailsListQuery.graphql"
+
+export interface PartnerArtistDetailsListProps {
+  edges: PartnerArtistDetailsList_edges
+}
+
+// TODO: Rewrite to using infinite scroll
+export const PartnerArtistDetailsList: React.FC<PartnerArtistDetailsListProps> = ({
+  edges,
+}) => {
+  return (
+    <Box mt={4}>
+      {edges
+        .filter(artist => artist.isDisplayOnPartnerProfile)
+        .filter(artists => artists.counts.artworks > 0)
+        .map(edge => {
```

Comment:
Could just return early with these conditions in the `map` rather than calling `filter` twice and then a `map`

Reply:
I updated this component to support infinite scroll, could you please take a look again?